<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Radio3DRConfig</class>
 <widget class="QWidget" name="Radio3DRConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;h2&gt;3DR Radio&lt;/h2&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>127</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>125</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>149</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload Firmware</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset to Defaults</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="localGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>321</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Local</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="3">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>LBT RSSI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Max Freq.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="eccCheckBox">
      <property name="text">
       <string>ECC</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Net ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QComboBox" name="minFreqComboBox"/>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Node Dest.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QComboBox" name="nodeDestComboBox"/>
    </item>
    <item row="8" column="1">
     <widget class="QCheckBox" name="mavLinkCheckBox">
      <property name="text">
       <string>MAVLink</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QComboBox" name="nodeCountComboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Format</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Node ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QComboBox" name="numberChannelsComboBox"/>
    </item>
    <item row="9" column="1">
     <widget class="QCheckBox" name="opResendCheckBox">
      <property name="text">
       <string>Op Resend</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Air Baud</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Min Freq.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Tx Power</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string># Channels</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="netIdComboBox"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="formatLineEdit"/>
    </item>
    <item row="6" column="4">
     <widget class="QComboBox" name="lbtRssiComboBox"/>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="airBaudComboBox"/>
    </item>
    <item row="9" column="2" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Node Count</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="txPowerComboBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Baud</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QComboBox" name="maxFreqComboBox"/>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QTextEdit" name="rssiTextEdit">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QComboBox" name="dutyComboBox"/>
    </item>
    <item row="7" column="4">
     <widget class="QComboBox" name="nodeIdComboBox"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Version</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>versionLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Duty Cycle</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="baudComboBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>RSSI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="versionLineEdit"/>
    </item>
   </layout>
   <zorder>label_1</zorder>
   <zorder>versionLineEdit</zorder>
   <zorder>label_2</zorder>
   <zorder>formatLineEdit</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
   <zorder>minFreqComboBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>maxFreqComboBox</zorder>
   <zorder>baudComboBox</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>numberChannelsComboBox</zorder>
   <zorder>label_7</zorder>
   <zorder>dutyComboBox</zorder>
   <zorder>label_10</zorder>
   <zorder>lbtRssiComboBox</zorder>
   <zorder>label_11</zorder>
   <zorder>nodeIdComboBox</zorder>
   <zorder>label_12</zorder>
   <zorder>nodeDestComboBox</zorder>
   <zorder>label_13</zorder>
   <zorder>nodeCountComboBox</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>airBaudComboBox</zorder>
   <zorder>label_14</zorder>
   <zorder>netIdComboBox</zorder>
   <zorder>label_17</zorder>
   <zorder>txPowerComboBox</zorder>
   <zorder>eccCheckBox</zorder>
   <zorder>mavLinkCheckBox</zorder>
   <zorder>opResendCheckBox</zorder>
   <zorder>rssiTextEdit</zorder>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>651</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="remoteGroupBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>321</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Remote</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Net ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QComboBox" name="minFreqComboBox_2"/>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Node Dest.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QComboBox" name="nodeDestComboBox_2"/>
    </item>
    <item row="8" column="1">
     <widget class="QCheckBox" name="mavLinkCheckBox_2">
      <property name="text">
       <string>MAVLink</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QComboBox" name="nodeCountComboBox_2"/>
    </item>
    <item row="9" column="1">
     <widget class="QCheckBox" name="opResendCheckBox_2">
      <property name="text">
       <string>Op Resend</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Air Baud</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Min Freq.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QComboBox" name="numberChannelsComboBox_2"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Tx Power</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Node ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string># Channels</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="netIdComboBox_2"/>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="versionLineEdit_2"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="formatLineEdit_2"/>
    </item>
    <item row="6" column="4">
     <widget class="QComboBox" name="lbtRssiComboBox_2"/>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="airBaudComboBox_2"/>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>LBT RSSI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="txPowerComboBox_2"/>
    </item>
    <item row="5" column="4">
     <widget class="QComboBox" name="dutyComboBox_2"/>
    </item>
    <item row="7" column="4">
     <widget class="QComboBox" name="nodeIdComboBox_2"/>
    </item>
    <item row="9" column="2" colspan="2">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Node Count</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Baud</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QComboBox" name="maxFreqComboBox_2"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Version</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>Duty Cycle</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="baudComboBox_2"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>RSSI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>Format</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QTextEdit" name="rssiTextEdit_2"/>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Max Freq.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="eccCheckBox_2">
      <property name="text">
       <string>ECC</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy Required
Items to Remote</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>66</y>
     <width>131</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>Link Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="linkPortComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>123</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="baudPortComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>123</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>91</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="settingsButton">
      <property name="text">
       <string>Adv. Settings</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
